<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qrbox</class>
 <widget class="QWidget" name="qrbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qrbox</string>
  </property>
  <widget class="QLabel" name="labe_Qrcode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Qrstatus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
